### PR TITLE
README.md TAMIL

### PR DESCRIPTION
**Pull request by aasim-syed**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

**Comment by AsswinCR** [comment]
Hi, is it still open or are any changes to be made?

**Review by erictleung — CHANGES_REQUESTED** [review]
Thanks for making these changes! Sorry for the delay. Thanks for adding the progress GitHub Actions, but I think the "in progress" label should be added only if there is someone actively working on it or claimed it. So I would say remove that from the PR. Otherwise, I'll wait on someone else to verify your translations. I'm not qualified to check your translation.

File: README.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 <!-- Do not translate this -->
 <details>
 <summary>
-<strong> Read this guide in other languages </strong>
+<strong> இந்த Read.md மித மொழிகளில் உள்ளன  </strong>
```
Comment (synackred) [inline README.md:4]: change மற்ற i think that correct
இந்த வழிகாட்டியை மற்ற மொழிகளில் படிக்கவும்

Comment (erictleung) [reply README.md:4]:
```suggestion
<strong>இந்த வழிகாட்டியை மற்ற மொழிகளில் படிக்கவும்</strong>
```

File: .github/workflows/progress.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      - reopened
+      - opened
+jobs:
+  label_issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Label issues
+        uses: andymckay/labeler@5c59dabdfd4dd5bd9c6e6d255b01b9d764af4414
```
Comment (erictleung) [inline .github/workflows/progress.yml:14]:
```suggestion
        uses: andymckay/labeler@1.0.4
```

**Comment by erictleung** [comment]
Closing as this makes translations to the main README and it is stale. Happy coding!

**Pull request closed without merging**